src/Whoops/Util/TemplateHelper.php

```diff
@@ -57,6 +57,19 @@ public function escapeButPreserveUris($raw)
             "<a href=\"$1\" target=\"_blank\">$1</a>", $escaped
         );
     }
+	
+    /**
+     * Format the given value into a human readable string
+     *
+     * @param  mixed $value
+     * @return string
+     */
+    public function dump($value) {
+        if (class_exists('\Symfony\Component\VarDumper\VarDumper')) {
+            return \Symfony\Component\VarDumper\VarDumper::dump($value);
```

Comment:
maybe better to import this - importing doesn't fail if the class doesn't exist - infact php doesn't even attempt to autoload it at that point